biome.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
+  "vcs": {
+    "enabled": false,
+    "clientKind": "git",
+    "useIgnoreFile": false
+  },
+  "files": {
+    "ignoreUnknown": false,
+    "ignore": ["dist", "README", "LICENSE", "package.json", "tsconfig.json"]
+  },
+  "formatter": {
+    "enabled": true,
+    "indentStyle": "tab"
```

Comment:
```suggestion
    "indentStyle": "space"
```

Reply:
space > tab, it's simple

Reply:
+1 for @rafa-thayto. I like tabs, but space is more natural and consume less space on the screens

Reply:
Got it, thanks